realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/entities/SyncAllTypes.kt

```diff
@@ -104,6 +109,8 @@ open class SyncAllTypes : RealmObject() {
 
     @Required
     var columnObjectIdList: RealmList<ObjectId>? = null
+    @Required
+    var columnUUIDList: RealmList<UUID>? = null
```

Comment:
Having these lists nullable is just akward. Core will never return null for them, so it would be easier to just make them `var columnUUIDList: RealmList<UUID>`. Not sure why ObjectId was made nullable.

Reply:
They are actually defined like that for all lists, I followed through.